### PR TITLE
fix(ci): python version for pre-commit and fix mypy int casting

### LINKED ISSUE #62: bug(ci): ci is broken because of default python version in pre-commit
### Description

The CI is broken due to the default python version

### What Jumanji version are you using?

_No response_

### Which accelerator(s) are you using?

_No response_

### Additional System Info

_No response_

### Additional Context

_No response_

### (Optional) Suggestion

_No response_

### PR DESCRIPTION
**Pull request by clement-bonnet**
Resolves #62 

- fix(ci): default python version for pre-commit
- fix(ci): cast numpy int to int to solve mypy failing

### CODE REVIEW

File: jumanji/environments/combinatorial/routing/env_viewer.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def __init__(
 
         for _ in range(num_agents):
             color = rnd.randint(0, 192, 3)
+            color = list(map(int, color))
             self.palette.append((color[0], color[1], color[2]))
```
Comment (cyprienc) [inline jumanji/environments/combinatorial/routing/env_viewer.py:68]:
```suggestion
            self.palette.extend(map(int, color))
```

Comment (clement-bonnet) [reply jumanji/environments/combinatorial/routing/env_viewer.py:68]: Would have been simpler, but I get a mypy error.
```
jumanji/environments/combinatorial/routing/env_viewer.py:69: error: Argument 1 to "append" of "list" has incompatible type "Tuple[int, ...]"; expected "Tuple[int, int, int]"
```
Unless you see a way around it, I will stick to the slightly more verbose solution.

Comment (clement-bonnet) [reply jumanji/environments/combinatorial/routing/env_viewer.py:68]: I found a way around it based on your suggestion.
```
            r, g, b = map(int, rnd.randint(0, 192, 3))
            self.palette.append((r, g, b))
```
Agree to go with this?

**Comment by clement-bonnet** [comment]
Issue solved in #64

**Pull request closed without merging**